src/mumble/AudioOutput.ui

```diff
@@ -330,6 +330,36 @@
         </item>
        </layout>
       </item>
+      <item row="6" column="1">
+       <layout class="QHBoxLayout" name="horizontalLayout_2">
+        <item>
+         <widget class="QCheckBox" name="qcbOnlyAttenuateSameSink">
+          <property name="toolTip">
+           <string>If checked, Mumble will only attenuate applications that are using the same output source as Mumble</string>
+          </property>
+          <property name="whatsThis">
+           <string>&lt;b&gt;Attenuate only applications using the same output as Mumble&lt;/b&gt;&lt;br /&gt;If checked, applications that use a different output than Mumble will not be attenuated.</string>
+          </property>
+          <property name="text">
+           <string>Only same output</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QCheckBox" name="qcbAttenuateLoopbacks">
+          <property name="toolTip">
+           <string>PulseAudio only: If checked, loopback modules will be attenuated</string>
+          </property>
+          <property name="whatsThis">
+           <string>&lt;b&gt;PulseAudio only: Also attenuate loopback modules.&lt;/b&gt;&lt;br /&gt;If loopback modules are linked to Mumble's output/sink, they will also be attenuated.</string>
+          </property>
+          <property name="text">
+           <string>Include loopbacks</string>
```

Comment:
Maybe "Include PulseAudio loopbacks" or "Include PulseAudio loopback modules"?

I'm not sure on the terminology to use here, but "Include loopbacks" is a little vague if you do not include the context (PulseAudio) -- even if there's a tooltip and What's This.